terraform/cp4ba/variables.tf

```diff
@@ -70,36 +73,46 @@ variable "on_vpc" {
   description = "Select 'true' to install on a VPC cluster and it's using VPC Gen2. Note: CP4BA does not currently support VPC cluster."
```

Comment:
Remove description for `on_vpc` from the README.  It should be commented out in the `variables.tf` file.

Reply:
Fixed